cloud_functions/climateiq_export_to_aws_cf/main.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime
+import os
+
+import boto3
+import flask
+import functions_framework
+from google.cloud import secretmanager
+from google.cloud.storage import client as gcs_client
+
+# GCS bucket name, where merged prediction outputs are stored.
+GCS_BUCKET_NAME = (
+    os.environ.get("BUCKET_PREFIX", "") + "climateiq-spatialized-merged-predictions"
+)
+# AWS bucket name, where to copy files to.
+S3_BUCKET_NAME = "climateiq-data-delivery"
+
+# IDs for retrieving secrets to authenticate to AWS
+PROJECT_ID = "climateiq"
```

Comment:
this will be variable depending on if we are running in prod or dev. for prod, os.environ.get("BUCKET_PREFIX", "") will be "" and for dev, os.environ.get("BUCKET_PREFIX", "") will be "test-"

Reply:
(the dev GCP project is climateiq-test)